zowe-rest-api-sample-spring/src/main/jcl/template.jcl

```diff
@@ -58,7 +58,11 @@ IJO="-Xms16m -Xmx128m"
 IJO="$IJO -Dibm.serversocket.recover=true"
 IJO="$IJO -Dfile.encoding=UTF-8"
 IJO="$IJO -Djava.io.tmpdir=/tmp"
-#IJO="$IJO -Xrunjdwp:transport=dt_socket,server=y,address=24621"
+{{#if debugPort}}
```

Comment:
i ❤️ handlebars

Reply:
I have recommended Handlebars to one team during the Hackathon